### PR TITLE
Add disk group for mdev zvol enumeration

### PR DESCRIPTION
**Pull request by andrewd-zededa**
Resolve regression from commit 3f78153.  

Touching /dev/mdev.log would show mdev errors like the below:

mdev[2051]: unknown user/group 'root:disk' on line 7

### CODE REVIEW

File: pkg/dom0-ztools/Dockerfile [hunk 1]
```diff
@@ -58,3 +58,5 @@ RUN addgroup -S tpms
 RUN addgroup -S vtpm
 RUN adduser -S -D -H -h /nonexistent -s /bin/false -g "vtpm" -G vtpm vtpm
 RUN addgroup vtpm tpms
+
```
Comment (shjala) [inline pkg/dom0-ztools/Dockerfile:61]: would you please add comment about the group?

**Review by eriknordmark — APPROVED** [review]
LGTM but please add the comment @shjala asked for.

File: pkg/pillar/Dockerfile [hunk 2]
```diff
@@ -10,7 +10,7 @@ ARG BUILD_PKGS_BASE="git gcc linux-headers libc-dev make linux-pam-dev m4 findut
 # we use the same image in several places
 ARG EVE_ALPINE_IMAGE=lfedge/eve-alpine:1f7685f95a475c6bbe682f0b976f12180b6c8726
 
-FROM lfedge/eve-dom0-ztools:81a7af28b32c7558e1c4622e60a9e3197075decd as zfs
```
Comment (eriknordmark) [inline pkg/pillar/Dockerfile:13]: This reference is causing build failures.

**Review by eriknordmark — CHANGES_REQUESTED** [review]

File: pkg/dom0-ztools/Dockerfile (hunk 1 above)
Comment (shjala) [reply pkg/dom0-ztools/Dockerfile:61]: now that #3989 is merged, please rebase and use a fixed id for the group.

Comment (andrewd-zededa) [reply pkg/dom0-ztools/Dockerfile:61]: Yeah I'll add both requests

**Comment by andrewd-zededa** [comment]
```
ERROR: failed to solve: golang:1.20.1-alpine: failed to copy: httpReadSeeker: failed open: unexpected status code https://registry-1.docker.io/v2/library/golang/manifests/sha256:87d0a3309b34e2ca732efd69fb899d3c420d3382370fd6e7e6d2cb5c930f27f9: 429 Too Many Requests - Server message: toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
make: *** [Makefile:944: eve-build-ubuntu] Error 1
Error: Process completed with exit code 2.
```

too many pulls on latest build...

**Comment by andrewd-zededa** [comment]
Added a yetus header complaint fix in pkg/pillar/build-kubevirt.yml so that I could kick the build again.

**Review by eriknordmark — APPROVED** [review]
Run again

File: pkg/pillar/Dockerfile (hunk 2 above)
Comment (eriknordmark) [reply pkg/pillar/Dockerfile:13]: This is still failing:
Dockerfile:13
--------------------
  11 |     ARG EVE_ALPINE_IMAGE=lfedge/eve-alpine:1f7685f95a475c6bbe682f0b976f12180b6c8726
  12 |     
  13 | >>> FROM lfedge/eve-dom0-ztools:f923d706814267c16715ed2db64f289bc479c8d4 as zfs

**Comment by andrewd-zededa** [comment]
Pushed a rebase

**Comment by andrewd-zededa** [comment]
newline on the yml file for yetus

**Comment by eriknordmark** [comment]
> newline on the yml file for yetus

What about the build failure in
https://github.com/lf-edge/eve/actions/runs/9668949206/job/26674312202?pr=3996

**Comment by andrewd-zededa** [comment]
yeah I think the tag changed again, must have been during my rebase.  One more try

**Comment by andrewd-zededa** [comment]
I'm waiting for more builds to complete locally, the go tests failure is odd

**Comment by eriknordmark** [comment]
> I'm waiting for more builds to complete locally, the go tests failure is odd

Did you try a make test  in your workspace?

**Comment by andrewd-zededa** [comment]
reverted the pkg/pillar/Dockerfile back to origin/master.  Will push in separate pr.

**Review by eriknordmark — APPROVED** [review]
LGTM

**Comment by andrewd-zededa** [comment]
> > I'm waiting for more builds to complete locally, the go tests failure is odd
> 
> Did you try a make test in your workspace?

yeah, it failed for not having a local containerd to run against but ran many other tests.  It didn't fail due to the dom0-ztools tag.

**Comment by andrewd-zededa** [comment]
lf-edge/eden get-run-id failure (pull_request_review)

```
Error: No successful runs found
    at eval (eval at callAsyncFunction (/home/runner/work/_actions/actions/github-script/v7/dist/index.js:35424:16), <anonymous>:11:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async main (/home/runner/work/_actions/actions/github-script/v7/dist/index.js:35522:20)
Error: Unhandled error: Error: No successful runs found
```

I'm not sure what to make of this failure

File: pkg/pillar/Dockerfile (hunk 2 above)
Comment (andrewd-zededa) [reply pkg/pillar/Dockerfile]: removed pkg/pillar/Dockerfile from PR, will push afterwards

**Comment by andrewd-zededa** [comment]
> lf-edge/eden get-run-id failure (pull_request_review)
> 
> ```
> Error: No successful runs found
>     at eval (eval at callAsyncFunction (/home/runner/work/_actions/actions/github-script/v7/dist/index.js:35424:16), <anonymous>:11:11)
>     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
>     at async main (/home/runner/work/_actions/actions/github-script/v7/dist/index.js:35522:20)
> Error: Unhandled error: Error: No successful runs found
> ```
> 
> I'm not sure what to make of this failure

never mind, it switched to skipped after resolving the pkg/pillar/Dockerfile changes needed.  If this is merged I'll submit a PR to bump dom0-ztools tag in pillar.

**Review by eriknordmark — APPROVED** [review]
Run again

**Pull request merged**